docs/ValidationRules/RULEID.RULEFRIENDLYNAME.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+/*
+ * INSTRUCTIONS:
+ * 
+ * - Replace `RULEID` with a valid value.
+ *      It should start with a prefix "SARIF", followed by 
+ *      a 4 digit number which is the next sequential number available
+ *      to use in ~\src\Sarif.Multitool\Rules\RuleId.cs file.
+ *      Example:
+ *          SARIF1023
+ *
+ * - Replace `RULEFRIENDLYNAME` with a valid value.
+ *      RULEFRIENDLYNAME should concisely define the purpose of this rule.
+ *      Use imperative language, like `UseAbsoluteUri`
```

Comment:
> Use imperative language, like `UseAbsoluteUri [](start = 7, length = 46)

i changed the language here to be stricter.

Reply:
good.

---
In reply to: [435587873](https://github.com/microsoft/sarif-sdk/pull/1902#discussion_r435587873) [](ancestors = 435587873)